src/components/UserCommentCard.jsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import Grid from "@mui/material/Grid";
+import dayjs from "dayjs";
+import { Button } from "@mui/material";
+import { deleteCommentById, getArticleComments } from "../api/api";
+import { useState } from "react";
+
+export default function UserCommentCard({ comment, setComments, articleID }) {
+  const [isActivated, setIsActivated] = useState(false);
+
+  const handleClick = () => {
+    if (!isActivated) {
+      deleteCommentById(comment.comment_id).then(() => {
+        getArticleComments(articleID).then(({ comments }) => {
+          setComments(comments);
+        });
```

Comment:
just to mention - because you are making a fresh request for comments the user might see a lot of confusing updates to the page when they have only clicked a delete button